packages/estatico-boilerplate/src/assets/js/helpers/app.js

```diff
@@ -37,8 +37,8 @@ class App {
 
   initModule(moduleName, element) {
     const Module = window[namespace].modules[moduleName].Class;
-    const metaData = element.dataset[`${moduleName}Data`] || {};
-    const metaOptions = element.dataset[`${moduleName}Options`] || {};
+    const metaData = this.parseData(element, `${moduleName}Data`);
```

Comment:
@orioltf, FYI: I had to fix the data attribute parsing, discovered it while playing around with the slideshow demo.